src/ActionModal/components/ActionModal.vue

```diff
@@ -415,6 +415,11 @@ export default {
     ...mapGetters([`connected`, `isExtensionAccount`]),
     ...mapGetters({ networkId: `network` }),
     checkFeatureAvailable() {
+      /* istanbul ignore next */
+      if (this.network.testnet) {
```

Comment:
no no no no ^^

Reply:
```suggestion
      if (this.network.id === 'local-cosmos-hub-testnet') {
```

Reply:
`gaia-13006` is also a `testnet`

Reply:
It would be the best option if this would be solved via the API I think

Reply:
Good point. I remember we discussed this the other day too. 👍